pkg/session_recording/recordings.jsx

```diff
@@ -349,10 +420,29 @@
             cockpit.location.go([recording.id], cockpit.location.options);
         }
 
+        componentDidUpdate() {
+            this.drawSortArrow(this.state.sorting_field);
+        }
+
         render() {
-            let columnTitles = [_("User"), _("Start"), _("End"), _("Duration")];
+            let columnTitles = [
+                (<div className="sort" onClick={this.sortByUser}><span>{_("User")}</span> <div ref="user"
+                    className="sort_icon"></div></div>),
+                (<div className="sort" onClick={this.sortBySince}><span>{_("Start")}</span> <div ref="start"
+                    className="sort_icon"></div></div>),
+                (<div className="sort" onClick={this.sortByUntil}><span>{_("End")}</span> <div ref="end"
+                    className="sort_icon"></div></div>),
+                (<div className="sort" onClick={this.sortByDuration}><span>{_("Duration")}</span> <div ref="duration"
```

Comment:
Can we avoid having all these separate handler functions? Perhaps have a single handler and inspect the element being clicked on? Perhaps get the clicked element ID and deduce the column to sort from it?